react-toggle-switch-component/my-app/src/ToggleSwitch.js

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+
+export default function ToggleSwitch() {
+  const [toggle, setToggle] = useState('false');
+  const [switchText, setSwitchText] = useState('OFF');
```

Comment:
I wouldn't save this to state, it's causing this to re-render more times than needed. I would just use a ternary to check if `toggle` is truthy or not, and assign the variable a value that way.

`const switchText = toggle ? 'ON' : 'OFF'`